### ISSUE TITLE
Cell to cell navigation in `CellEdit` should include `WidgetColumn` cells with focusable content

### ISSUE THREAD

**Issue by JockeLindberg** [bryntum/support#6999]
Reproducable in Grid widgetcolumn demo:
1. Left/right arrow keys moves to new cell when cursor is in a field inside a widget column.
2. In a widget column containing multiple checkboxes there is no focus outline or indicator showing which checkbox is currently being focused.
3. When cellEditing and hitting TAB, focus should probably move to next focusable column. Now it moves to the next cell in which cellEdit can begin editing.

Should probably test more configurations of widget column.

**Comment by ghulamghousdev** [comment]
Another report here related to point 3 in the ticket description: https://forum.bryntum.com/viewtopic.php?p=162166#p162166

**Comment by FlowIT-JIT** [comment]
Note to self: This should also fix https://forum.bryntum.com/viewtopic.php?t=31821

**Comment by FlowIT-JIT** [comment]
Is there an ETA for this fix? It is dating back to June 2023. Would it be possible to assign a milestone?

**Comment by ExtAnimal** [comment]
> Reproducable in Grid widgetcolumn demo:
> 
> 1. Left/right arrow keys moves to new cell when cursor is in a field inside a widget column.

That does not hjappen:

https://github.com/user-attachments/assets/5a91df6e-e7a9-46fa-b5ed-28ad72568eee

**Comment by ExtAnimal** [comment]
> Is there an ETA for this fix? It is dating back to June 2023. Would it be possible to set a milestone?

Hopefully, there will be the `tabToFocusables` property in `CellEdit` in the next version:

<img width="904" alt="Image" src="https://github.com/user-attachments/assets/52cb98ac-d865-4c4f-9397-b75275070134" />
